packages/terra-action-header/src/_ActionHeaderContainer.jsx

```diff
@@ -57,14 +57,14 @@ const ActionHeaderContainer = ({
   ) : undefined;
 
   return (
-    <header {...customProps} className={cx(['flex-header', customProps.className])}>
+    <div {...customProps} className={cx(['flex-header', customProps.className])}>
       {startContent && <div className={cx('flex-end')}>{startContent}</div>}
       <div className={cx('flex-fill')}>
         {titleElement}
       </div>
       {content}
       {endContent && <div className={cx('flex-end')}>{endContent}</div>}
-    </header>
+    </div>
```

Comment:
I would assume a few snapshots are going to fail with this change

Reply:
I ran jest locally and didn't see any issues in terra-core, there may be some issues downstream in jest tests.